client/blocks/upsell-nudge/index.jsx

```diff
@@ -60,7 +60,17 @@ export const UpsellNudge = ( {
 		href = addQueryArgs( { feature, plan }, `/plans/${ site.slug }` );
 	}
 
-	return <Banner { ...props } showIcon={ showIcon } className={ classes } href={ href } />;
+	return (
+		<Banner
+			{ ...props }
```

Comment:
I think it'd be safer to be explicit here and pass down only whitelisted props so we don't use `...props` any more in the banner (which I believe was @gwwar's [suggestion](https://github.com/Automattic/wp-calypso/pull/41115/files#r408500126))

Reply:
Ohhh OK, gotcha. I'll do another pass and evaluate the other props and include those explicitly as well. Thanks!